src/components/FindWallet/WalletFilterSidebar/WalletFilterFeature/index.tsx

```diff
@@ -104,7 +104,7 @@ const WalletFilterFeature = ({
                     _hover={{ background: "transparent" }}
                   >
                     <Box as="span" flex={1}>
-                      {filterOption.title}
+                      <Text color="text">{filterOption.title}</Text>
```

Comment:
`text` is not a token from the DS. Would be nice if we can use one of the semantic tokens. Not a blocker.

Reply:
Fixed